packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/xdd/ExtractionResource.java

```diff
@@ -0,0 +1,72 @@
+package software.uncharted.terarium.hmiserver.resources.xdd;
+
+import javax.inject.Inject;
+import javax.ws.rs.POST;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+
+import software.uncharted.terarium.hmiserver.models.xdd.Extraction;
+import software.uncharted.terarium.hmiserver.models.xdd.XDDExtractionsResponseOK;
+import software.uncharted.terarium.hmiserver.models.xdd.XDDResponse;
+import software.uncharted.terarium.hmiserver.proxies.xdd.ExtractionProxy;
+import software.uncharted.terarium.hmiserver.services.xdd.DocumentService;
+
+@Path("/api/xdd/extractions")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Tag(name = "XDD Extraction REST Endpoint")
+public class ExtractionResource {
+
+		@Inject
+		DocumentService documentService;
+
+		@RestClient
+		ExtractionProxy proxy;
+
+		@GET
+		@Consumes(MediaType.TEXT_PLAIN)
+		@Produces(MediaType.APPLICATION_JSON)
+		@Tag(name = "Search XDD for extractions related to the document identified in the payload")
+		public Response searchExtractions(@QueryParam("doi") String doi) {
+			final List<Extraction> extractions = new ArrayList<>();
```

Comment:
remove all this and just return the response from the proxy
`return proxy.getExtractions(doi);`

Reply:
done